### PR TITLE
Validate oci-layout version

### PR DESCRIPTION
**Pull request by coolljt0725**
OCI image specification has required that
`"oci-layout" MUST contain a JSON object with a version field {"imageLayoutVersion": "1.0.0"}`,
the image tool should follow this rule.

Signed-off-by: Lei Jitang leijitang@huawei.com

### CODE REVIEW

File: image/image.go [hunk 1]
```diff
@@ -49,6 +58,9 @@ var validRefMediaTypes = []string{
 }
 
 func validate(w walker, refs []string, out *log.Logger) error {
+	if err := validatImageLayout(w); err != nil {
```
Comment (wking) [inline image/image.go]: `validatImageLayout` → `validateImageLayout`.

**Comment by vbatts** [comment]
I'm +1, but `make lint` failure needs to be fixed.

**Comment by coolljt0725** [comment]
wait for https://github.com/opencontainers/image-spec/pull/519 and then rebase

**Comment by coolljt0725** [comment]
close by https://github.com/opencontainers/image-tools/issues/84

**Pull request closed without merging**